src/ClassApplication/Controllers/ClassController.cs

```diff
@@ -40,11 +41,89 @@ public async Task<ActionResult<string>> CreateClassroom(Classroom classroom)
         /// <param name="id">classroom id</param>
         /// <returns>An array containing a value determined by the parameter</returns>
         [HttpGet("GetClassByID/{id}")]
-        public async Task<ActionResult<string>> GetClassroom(string id)
+        /*
+                public async Task<ActionResult<string>> GetClassroom(string id)
+                {
+                    //to check if i am pushing it right
+                    //throw new NotImplementedException();
+         }
+         */
+        public async Task<ActionResult<string>> GetClassroomByID(string id)
         {
-            throw new NotImplementedException();
+
+
+            string Text = " SELECT * FROM c WHERE c.noteId =@id";

```

Comment:
So in this case, because were searching by id, we can expect each class to have a unique id. So instead of querying, we could do a classesContainer.ReadItemAsync<>();

Reply:
True, I believe i  fixed it.

Reply:
Do you think it is the same case for getClassRoomByName also ?